.travis.gogenerate.sh

```diff
@@ -1,11 +1,10 @@
 #!/bin/bash
 
-if [[ "$TRAVIS_GO_VERSION" =~ ^1\.[45](\..*)?$ ]]; then
+if [[ "$TRAVIS_GO_VERSION" =~ ^1\.(12|13)(\..*)?$ ]]; then
   exit 0
 fi
 
-go get github.com/ernesto-jimenez/gogen/imports
-go generate ./...
+GO111MODULE=on go generate ./...
```

Comment:
Wouldn't this override the travis `env` setting? Should this line not just be `go generate ./...`?

Reply:
Yup I'll change how this works. Thanks.

Reply:
Updated in 1f912b6.